src/common/mini_pytor/client.py

```diff
@@ -426,9 +415,9 @@ def req(request, intermediate_relays):
             print("socketerror")
```

Comment:
use `print("socket error", file=sys.stderr)` instead so that it goes to stderr instead of the default stdout.